package org.finra.herd.rest;

import org.junit.Assert;
import org.junit.rules.ExpectedException;
import org.junit.Test;

import java.lang.reflect.Array;
import java.lang.reflect.Constructor;
import java.lang.reflect.InvocationTargetException;
import java.lang.reflect.Method;

import com.diffblue.deeptestutils.CompareWithFieldList;
import com.diffblue.deeptestutils.Reflector;

@org.junit.runner.RunWith(org.powermock.modules.junit4.PowerMockRunner.class)
public class BusinessObjectDataRestControllerTestCase {

  @org.junit.Rule
  public ExpectedException thrown = ExpectedException.none();

  /* testedClasses: BusinessObjectDataRestController */


  /*
   * Test generated by Diffblue Deeptest.
   * This test case covers the entire method.
   */

  @Test
  public void org_finra_herd_rest_BusinessObjectDataRestController__init__000_38b84d77c014ab49() throws Throwable {

    // Act, creating object to test constructor
    org.finra.herd.rest.BusinessObjectDataRestController instance = new org.finra.herd.rest.BusinessObjectDataRestController();

    // Method returns void, testing that no exception is thrown

  }
}

